### PR TITLE
Add `simplecov` and `codecov` as dev dependencies

### PR DESCRIPTION
**Pull request by botandrose**
We want to use this tools to track code coverage and display it in our README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 [![Build Status](https://travis-ci.org/DatabaseCleaner/database_cleaner-active_record.svg?branch=master)](https://travis-ci.org/DatabaseCleaner/database_cleaner-active_record)
 [![Code Climate](https://codeclimate.com/github/DatabaseCleaner/database_cleaner-active_record/badges/gpa.svg)](https://codeclimate.com/github/DatabaseCleaner/database_cleaner-active_record)
+[![codecov](https://codecov.io/gh/DatabaseCleaner/database_cleaner/branch/master/graph/badge.svg)](https://codecov.io/gh/DatabaseCleaner/database_cleaner-active_record)
```
Comment (etagwerker) [inline README.md:5]: @botandrose I think the link should be different here, right?

Comment (botandrose) [reply README.md:5]: Ah, good catch!

**Comment by botandrose** [comment]
@etagwerker Thanks for taking a look and pointing me in the right direction! I just added `COVERAGE=true` to Travis, and that seems to have kicked things off. Regarding the CodeCov token, the docs said "Not required on Travis-CI, CircleCI, AppVeyor, or Azure Pipelines for public repositories." so I'm gonna see if I can get away without it. :)

**Comment by botandrose** [comment]
Looking good! Gonna merge. Thanks for your help, @etagwerker !

**Pull request merged**